pysat/_instrument.py

```diff
@@ -3023,11 +3027,6 @@ def remote_file_list(self, start=None, stop=None):
         return a subset of available files.
 
         """
-        # Set the user-supplied kwargs
-        if 'list_remote_files' in self.kwargs.keys():
-            kwargs = self.kwargs['list_remote_files']
-        else:
-            kwargs = {}
```

Comment:
Removed this since it reset the kwargs with the defaults rather than user-supplied options.

Reply:
The problem with talking this out is that it means that the user/password that you can set on instrument instantiation are no longer applied here.  This needs to be changed to allow that, not removed.

Reply:
I'm making a change and testing it.